frontend/src/components/authentication/ForgotPassword.tsx

```diff
@@ -0,0 +1,90 @@
+import React, {useState, useCallback} from 'react'
+import {useDispatch} from 'react-redux'
+import {CssBaseline, TextField} from '@material-ui/core'
+import {Button, Avatar, Typography, FormControlLabel, Checkbox, Link, Paper, Grid }  from '@material-ui/core'
```

Comment:
Remove not used variables and components

Reply:
Done.